src/frame-archive.h

```diff
@@ -225,4 +227,4 @@ namespace librealsense
 
     };
 
-}
\ No newline at end of file
+}
```

Comment:
Can you also add a check that a source file will always end with an empty line (like this one) ?